src/vue-pdf-embed.vue

```diff
@@ -13,6 +20,14 @@ pdf.GlobalWorkerOptions.workerPort = new PdfWorker()
 export default {
   name: 'VuePdfEmbed',
   props: {
+    /**
+     * Id of the document to display 
+     * @values String
+     */
+    documentId: {
+      type: String,
+      required: false,
```

Comment:
Default value should be set for each props that are not marked as `required` (https://eslint.vuejs.org/rules/require-default-prop.html)

```suggestion
      default: null,
```